src/test/scala/services/ArticleFinderSpec.scala

```diff
@@ -34,8 +30,286 @@ class ArticleFinderSpec extends AnyWordSpec with Matchers {
           TestUtils.readJson(s"/exampleWithMultipleBodyPanelAndTabular.json").as[OctopusBundle]
         val expectedArticle = TestUtils.readJson(s"/article.json").as[OctopusArticle]
 
-        assert(ArticleFinder.findBodyText(articleWithMultipleBodyTexts) == Some(expectedArticle))
-        assert(ArticleFinder.findBodyText(articleWithBodyPanelAndTabularTexts) == Some(expectedArticle))
+        assert(ArticleFinder.findBodyText(articleWithMultipleBodyTexts).contains(expectedArticle))
+        assert(ArticleFinder.findBodyText(articleWithBodyPanelAndTabularTexts).contains(expectedArticle))
+      }
+      "prefer both to web and print" in {
```

Comment:
It may be good to move some of this setup into helper methods but it's not a blocker.

Reply:
Yeah, you're definitely right. It'll make it easier to understand the purpose of the test inputs.